### PR TITLE
feat: changed datastore setter methods to x_entry

### PR DESCRIPTION
**Pull request by LawMixer**
In this pull request, I've changed the ``get()`` to ``get_entry()`` for extra readability.

### CODE REVIEW

File: rblxopencloud/__init__.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import requests
 
 VERSION: str = "2.0.0"
-VERSION_INFO: Literal['alpha', 'beta', 'final'] = "alpha"
+VERSION_INFO: Literal['alpha', 'beta', 'final'] = "beta"
```
Comment (treeben77) [inline rblxopencloud/__init__.py:28]: I'm not sure if we want to move into beta yet.

File: rblxopencloud/http.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def send_request(method: str, path: str, authorization: Optional[str]=None,
     else:
         body = response.text
 
-    if DEBUG:
+    if DEBUG == True:
```
Comment (treeben77) [inline rblxopencloud/http.py:121]: I've changed this to `VERSION_INFO == "alpha"` in uncommitted changes but for now it is okay.

**Review by treeben77 — CHANGES_REQUESTED** [review]
everything's good except I believe we're not ready for version bump to beta.

File: rblxopencloud/__init__.py (hunk 1 above)
Comment (LawMixer) [reply rblxopencloud/__init__.py]: This has been solved.

**Review by treeben77 — APPROVED** [review]
everything looks good now

**Pull request merged**